[2025-04-19T11:40:16.227+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-19T11:40:16.283+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dbt_etl_pipeline_seed.dbt_seed manual__2025-04-18T00:33:16.135976+00:00 [queued]>
[2025-04-19T11:40:16.295+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dbt_etl_pipeline_seed.dbt_seed manual__2025-04-18T00:33:16.135976+00:00 [queued]>
[2025-04-19T11:40:16.296+0000] {taskinstance.py:2867} INFO - Starting attempt 3 of 3
[2025-04-19T11:40:16.311+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): dbt_seed> on 2025-04-18 00:33:16.135976+00:00
[2025-04-19T11:40:16.319+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=685) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-19T11:40:16.318+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'dbt_etl_pipeline_seed', 'dbt_seed', 'manual__2025-04-18T00:33:16.135976+00:00', '--job-id', '145', '--raw', '--subdir', 'DAGS_FOLDER/trigger_dbt.py', '--cfg-path', '/tmp/tmps0mu5j3l']
[2025-04-19T11:40:16.321+0000] {standard_task_runner.py:72} INFO - Started process 686 to run task
[2025-04-19T11:40:16.321+0000] {standard_task_runner.py:105} INFO - Job 145: Subtask dbt_seed
[2025-04-19T11:40:16.606+0000] {task_command.py:467} INFO - Running <TaskInstance: dbt_etl_pipeline_seed.dbt_seed manual__2025-04-18T00:33:16.135976+00:00 [running]> on host 446741231bc9
[2025-04-19T11:40:16.801+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='azizjon' AIRFLOW_CTX_DAG_ID='dbt_etl_pipeline_seed' AIRFLOW_CTX_TASK_ID='dbt_seed' AIRFLOW_CTX_EXECUTION_DATE='2025-04-18T00:33:16.135976+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-18T00:33:16.135976+00:00'
[2025-04-19T11:40:16.801+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-19T11:40:16.816+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-04-19T11:40:16.817+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', '\n        export PATH=/home/airflow/.local/bin:$PATH\n        cd /opt/airflow/commerce_dbt\n        dbt seed --profiles-dir .\n        ']
[2025-04-19T11:40:16.824+0000] {subprocess.py:99} INFO - Output:
[2025-04-19T11:40:19.183+0000] {subprocess.py:106} INFO - [0m11:40:19  Running with dbt=1.7.0
[2025-04-19T11:40:19.436+0000] {subprocess.py:106} INFO - [0m11:40:19  Registered adapter: postgres=1.7.0
[2025-04-19T11:40:19.921+0000] {subprocess.py:106} INFO - [0m11:40:19  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-04-19T11:40:19.922+0000] {subprocess.py:106} INFO - There are 1 unused configuration paths:
[2025-04-19T11:40:19.923+0000] {subprocess.py:106} INFO - - seeds.initial_dataset
[2025-04-19T11:40:19.972+0000] {subprocess.py:106} INFO - [0m11:40:19  Found 13 models, 1 snapshot, 6 seeds, 43 tests, 6 sources, 0 exposures, 0 metrics, 401 macros, 0 groups, 0 semantic models
[2025-04-19T11:40:19.975+0000] {subprocess.py:106} INFO - [0m11:40:19
[2025-04-19T11:40:20.588+0000] {subprocess.py:106} INFO - [0m11:40:20  Concurrency: 4 threads (target='dev')
[2025-04-19T11:40:20.591+0000] {subprocess.py:106} INFO - [0m11:40:20
[2025-04-19T11:40:20.606+0000] {subprocess.py:106} INFO - [0m11:40:20  1 of 6 START seed file commerce_raw.customers .................................. [RUN]
[2025-04-19T11:40:20.607+0000] {subprocess.py:106} INFO - [0m11:40:20  2 of 6 START seed file commerce_raw.order_items ................................ [RUN]
[2025-04-19T11:40:20.609+0000] {subprocess.py:106} INFO - [0m11:40:20  3 of 6 START seed file commerce_raw.orders ..................................... [RUN]
[2025-04-19T11:40:20.613+0000] {subprocess.py:106} INFO - [0m11:40:20  4 of 6 START seed file commerce_raw.product_categories ......................... [RUN]
[2025-04-19T11:40:21.554+0000] {subprocess.py:106} INFO - [0m11:40:21  4 of 6 OK loaded seed file commerce_raw.product_categories ..................... [[32mINSERT 4[0m in 0.89s]
[2025-04-19T11:40:21.578+0000] {subprocess.py:106} INFO - [0m11:40:21  5 of 6 START seed file commerce_raw.products ................................... [RUN]
[2025-04-19T11:40:22.290+0000] {subprocess.py:106} INFO - [0m11:40:22  5 of 6 OK loaded seed file commerce_raw.products ............................... [[32mINSERT 17[0m in 0.70s]
[2025-04-19T11:40:22.329+0000] {subprocess.py:106} INFO - [0m11:40:22  6 of 6 START seed file commerce_raw.stores ..................................... [RUN]
[2025-04-19T11:40:22.937+0000] {subprocess.py:106} INFO - [0m11:40:22  6 of 6 OK loaded seed file commerce_raw.stores ................................. [[32mINSERT 5[0m in 0.57s]
[2025-04-19T11:40:23.296+0000] {subprocess.py:106} INFO - [0m11:40:23  1 of 6 OK loaded seed file commerce_raw.customers .............................. [[32mINSERT 1000[0m in 2.67s]
[2025-04-19T11:40:32.782+0000] {subprocess.py:106} INFO - [0m11:40:32  3 of 6 OK loaded seed file commerce_raw.orders ................................. [[32mINSERT 7000[0m in 12.16s]
[2025-04-19T11:40:34.720+0000] {subprocess.py:106} INFO - [0m11:40:34  2 of 6 OK loaded seed file commerce_raw.order_items ............................ [[32mINSERT 10000[0m in 14.11s]
[2025-04-19T11:40:34.779+0000] {subprocess.py:106} INFO - [0m11:40:34
[2025-04-19T11:40:34.780+0000] {subprocess.py:106} INFO - [0m11:40:34  Finished running 6 seeds in 0 hours 0 minutes and 14.80 seconds (14.80s).
[2025-04-19T11:40:34.809+0000] {subprocess.py:106} INFO - [0m11:40:34
[2025-04-19T11:40:34.810+0000] {subprocess.py:106} INFO - [0m11:40:34  [32mCompleted successfully[0m
[2025-04-19T11:40:34.811+0000] {subprocess.py:106} INFO - [0m11:40:34
[2025-04-19T11:40:34.811+0000] {subprocess.py:106} INFO - [0m11:40:34  Done. PASS=6 WARN=0 ERROR=0 SKIP=0 TOTAL=6
[2025-04-19T11:40:35.875+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-04-19T11:40:36.006+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-19T11:40:36.007+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=dbt_etl_pipeline_seed, task_id=dbt_seed, run_id=manual__2025-04-18T00:33:16.135976+00:00, execution_date=20250418T003316, start_date=20250419T114016, end_date=20250419T114036
[2025-04-19T11:40:36.174+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-19T11:40:36.251+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-19T11:40:36.256+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-04-19T11:40:36.319+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
